components/Dropdown.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { View } from "react-native";
+import { useTheme } from "react-native-paper";
+import { Picker } from "@react-native-community/picker";
+
+export default function Dropdown({
```

Comment:
This component should probably be tested on iOS, I am afraid it might look strange there (unless you somehow set it to look as on Android)